modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -17,31 +16,34 @@ class NearbyController < ApplicationController
       skip_before_action(:authenticate)
       before_action :set_default_format
       before_action :set_facility_type
+      before_action :set_default_drivetime
       before_action :validate_params, only: [:index]
 
+      PER_PAGE = 20
+
       REQUIRED_PARAMS = {
```

Comment:
Out of scope/tech debt thought: Should we use Rails Strong Parameters here instead of a constant?

Reply:
Made a ticket for it.